apps/www/src/components/ip-allowlist-banner.tsx

```diff
@@ -0,0 +1,25 @@
+import { useIpAllowlist } from 'lib/context/IpAllowlistContext'
+import { useMe } from 'lib/context/MeContext'
+
+export function IpAllowlistBanner() {
+  const { allowlistName, hasAllowlistAccess } = useIpAllowlist()
+  const { isMember } = useMe()
+
+  // Don't show banner for users with member access or without allowlist access
+  if (!hasAllowlistAccess || isMember) {
```

Comment:
Also this banner should probably wait until both the allowlist and me query have loaded, otherwise it may show up unexpectedly (ie. when allowlist query has loaded and access is true but me query is still loading and then resolves to a member)